src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Serialization/ObjectWriter.cs

```diff
@@ -145,14 +150,34 @@ public void WriteValue(object? value)
             // Perf: Note that JIT optimizes each expression value.GetType() == typeof(T) to a single register comparison.
             // Also the checks are sorted by commonality of the checked types.
 
-            // The primitive types are
-            // Boolean, Byte, SByte, Int16, UInt16, Int32, UInt32,
-            // Int64, UInt64, IntPtr, UIntPtr, Char, Double, and Single.
+            // The list supported is from CConst.TryCreate:
+#if false
+        Friend Shared Function TryCreate(value As Object) As CConst
```

Comment:
>Friend Shared Function TryCreate(value As Object) As CConst [](http://example.com/codeflow?start=8&length=59)

the comment mentioning the location is good enough for me. I trust you enough that the #if false region gets in the way more than it's worth to me.

Reply:
done.